app/assets/stylesheets/layouts/_navbar.css.scss

```diff
@@ -112,3 +127,4 @@
 .navbar .btn-sign-in {
 	margin-right: 0px;
```

Comment:
Would you mind fixing the indentation here? (I know it wasn't changed in the PR, but if we don't know it'll probably never happen)